src/pages/support.tsx

```diff
@@ -8,14 +8,17 @@ import {
 } from "@chakra-ui/react";
 import { ChakraNextImage } from "components/Image";
 import { AppLayout } from "components/app-layouts/app";
-import { ContactSupportModal } from "components/help/contact-support-modal";
 import { NeedSomeHelp } from "components/help/need-some-help";
 import { NextSeo } from "next-seo";
+import dynamic from "next/dynamic";
 import { PageId } from "page-id";
 import { FiArrowRight } from "react-icons/fi";
 import { Card, Heading, Link, LinkButton, Text } from "tw-components";
 import { ThirdwebNextPage } from "utils/types";
 
+const ContactSupportModal = dynamic(
+  () => import("components/help/contact-support-modal"),
+);
```

Comment:
what's the benefit of lazy loading this?

Reply:
oops 💀 I meant to lazy load the other component (https://github.com/thirdweb-dev/dashboard/pull/2556/commits/c16a96a7bc8895e9457319c44e47b1da04bc11d3)

I will revert this